amundsen_application/static/js/components/TableDetail/NonEditableSection/index.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
```

Comment:
I would prefer to allow `EditableSection` to have a `readOnly` property that will just affect whether or not the edit button is displayed versus creating a whole new component that is exactly the same except for editing.

Reply:
ok sounds good, I struggled to get that to work, let me take another stab at it.

Reply:
figured out one possible solution I think!

Reply:
Great. Feel free to ping me on slack if you need further assistance on the React components.